.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: rust
+rust:
+  - 1.33.0
+  - stable
+  - beta
+  - nightly
+cache: cargo
```

Comment:
Caching `cargo` in Travis is known to be error prone and not necessarily speeding the builds up in an expected way. [Here](https://levans.fr/rust_travis_cache.html)'s a great write-up on the topic with a suggested fix.